sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blobs/Directory.Build.props

```diff
@@ -28,6 +28,6 @@
     <EnableClientSdkAnalyzers>false</EnableClientSdkAnalyzers>
 
     <!-- If there was no override specified, set the type of reference to use for client libraries from the Azure SDK repository -->
-    <UseProjectReferenceToAzureClients Condition="'$(UseProjectReferenceToAzureClients)' == ''">true</UseProjectReferenceToAzureClients>
+    <UseProjectReferenceToAzureClients Condition="'$(UseProjectReferenceToAzureClients)' == ''">false</UseProjectReferenceToAzureClients>
```

Comment:
@pakrym @chidozieononiwu should these properties be removed similar to the core/identity versions?

Reply:
They could, but I do not want them to be.   I want to be able to explicitly move them between project and package references - especially for the internal event hubs reference.   

I'd ask that we please leave them be.  This was a conscious decision and not an oversight during other changes.

Reply:
IMO they should be removed, everything should be a package reference now.

Reply:
I'm going to start a thread in teams about the different scenarios so we can figure out a strategy.